nonCisMaleInCs/DetailsView.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import {StyleSheet, View, Text, Image, Button, Alert, TouchableOpacity} from 'react-native';
+// import { Icon } from 'react-native-elements' 
+// import { Header } from 'react-native-elements';
+// import Icon from 'react-native-vector-icons/FontAwesome';
+import editIcon from './edit.png';
+import nophoto from './nophoto.png';
+
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+
+const onPressed = () => {
+  Alert.alert('Edit button got pressed!');
+};
+
+const DetailsView = () => {
+  return (
+    <>
+      <View style={styles.menuContainerStyle}>
+        <Text style={styles.menuText}>{'< Home'}</Text>
+      </View>
+      <View style={styles.headerImageContainer}>
+        <Image style={styles.imageStyle} source={nophoto} />
+      </View>
+      <View style={styles.body}>
+        <View style={styles.titleContainer}>
```

Comment:
Maybe we can get rid of the additional container wrapper for title and the following sections? What do you think? :)

Reply:
Done